package.json

```diff
@@ -15,7 +15,7 @@
     "required-version": "8.2.0.0-SNAPSHOT",
     "module-type": "templatesSet",
     "module-dependencies": "default",
-    "static-resources": "/assets,/css,/javascript,/icons,/img,/dist",
```

Comment:
I'm probably missing something here but I don't understand this change: what does the `/images` folder refer to exactly?

Reply:
/settings/template-thumbnail.png -> images/template-preview/template-thumbnail.png

https://github.com/Jahia/npm-modules-engine/blob/main/src/main/java/org/jahia/modules/npm/modules/engine/npmhandler/NpmProtocolConnection.java#L113-L116